store/mocks/Store.go

```diff
@@ -0,0 +1,103 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
```

Comment:
Can you lower case this filename

Reply:
pass to mockery -case underscore so all output files will be lowercase
```
-case string
        name the mocked file using casing convention [camel, snake, underscore] (default "camel")
```

Reply:
@asim @vtolstov 
Addressed in the new commit.